packages/@o3r/testing/schematics/ng-add/index.ts

```diff
@@ -36,7 +79,42 @@ export function ngAdd(options: NgAddSchematicsSchema): Rule {
         }),
         ngAddPeerDependencyPackages(['pixelmatch', 'pngjs'], testPackageJsonPath, dependencyType, options),
         registerPackageCollectionSchematics(packageJson)
-      ])(tree, context);
+      ];
+
+      if (installJest) {
+        if (workingDirectory !== undefined) {
+          const packageJsonFile = tree.readJson(`${workingDirectory}/package.json`) as PackageJson;
+          packageJsonFile.scripts ||= {};
+          packageJsonFile.scripts.test = 'jest';
+          tree.overwrite('package.json', JSON.stringify(packageJsonFile, null, 2));
```

Comment:
it is not **tree.overwrite(`${workingDirectory}/package.json`, ...**

Reply:
you are correct